certora/harness/EthereumVaultConnectorHarness.sol

```diff
@@ -33,7 +33,7 @@ contract EthereumVaultConnectorHarness is EthereumVaultConnector {
     }
 
     function getOwnerOf(bytes19 prefix) public view returns (address) {
-        return ownerLookup[prefix];
+        return ownerLookup[prefix].owner;
```

Comment:
This is the main part needed for proof repair after the updates. The rest is mainly related to upgrading the Certora-CLI (which seems to be forced on server side at this point)